### DISCUSSION TITLE
Does AqualinkD work without a factory pool conroller?

### DISCUSSION THREAD

**Discussion by Grauwulf75** [aqualinkd/AqualinkD#228] [Q&A]
Just wondering if I can use this project with my Jandy equipment without having a pre-existing pool controller? We just had our pool installed and I have a Jandy VS FloPro pump, Jandy JXi heater and a Jandy TruClear salt generator, but all operating in standalone mode, with no pool controller. Can I use AqualinkD to interface with all of the devices and then just use Home Assistant/Node Red to build my automations and remotely control my equipment?

I've been through all the documentation and it's not really spelled out if this is an option or not.

Thanks!

**Comment by sfeakes** [comment]
No this is not an option at this time.  AqualinkD connects to a jandy pool controller, and acts as a slave to that.
I started a project that would do what you are asking a number of years ago, but at that time I only had decoded SWG information, now I also have VSP, but I still need to work out JXi stuff.  And more importantly find the time. So it's probably not going to happen for a long time, or unless my Jandy controller craps out.

I also don't know of any other project that will do what you're asking, there is one for Pentair equipment though.  (link below).  This will control your SWG, since Pentair use the Jandy protocol for SWG. It could be modified to control your VSP, and I'm not sure on JXi since that's a protocol I have not decoded.
https://github.com/tagyoureit/nodejs-poolController

**Reply by Grauwulf75** [reply]
I appreciate the info! Any idea what model of Pentair SWG would be comparable with my Jandy unit to work with nodejs?

**Reply by sfeakes** [reply]
All SWG that I have come across support the Jandy protocol.  Even the Pentair SWG use Jandy protocol.  I know the link above to a nodejs pool controller can read jandy SWG, not sure if it can actually control it though. You’d had to post that question on that GitHub page.
